source/analytics/index.html.md.erb

```diff
@@ -201,6 +201,19 @@ After a change has been approved, it needs to be published. GTM will take you th
 
 You then need to manually update the default workspace by going to the default workspace, selecting **Submit** and then selecting **Update now** on the orange banner in the submit view. See the documentation on workspaces above for more details.
 
+### Adding custom dimensions to reflect your changes
+
+Custom dimensions is how GA recognises custom parameters set up in GTM tags. In order to see custom parameters from tags in GA event reporting, you need to set up custom dimension for each of your parameters. Only admins can update our custom dimensions so ask a team lead or a developer if you don't have the necessary permissions.
```

Comment:
Should the second "custom dimension" be either "a custom dimension" or "custom dimensions"?

(I'm not sure, I'm not really familiar with this stuff)

Reply:
It should be!